src/pack-n-test.ts

```diff
@@ -27,29 +27,46 @@ const writeFileP = promisify(writeFile);
 const tmpDirP = promisify(tmp.dir) as () => Promise<string>;
 const rimrafP = promisify(rimraf);
 
-export interface BaseCodeSample {
+export interface CodeSample {
+  js?: string;
```

Comment:
would it be possible to document the differences between the options (or why I would pick one over another)? Thinking specifically of cjs vs. js.

Reply:
That's a good point!

Reply:
@sofisl added a long-form explanation of the different properties in a code comment prior to the defined interface 😊  would love some feedback if you have some time to review it!